src/qt/checklst.cpp

```diff
@@ -80,8 +81,11 @@ bool wxCheckListBox::IsChecked(unsigned int n) const
 
 void wxCheckListBox::Check(unsigned int n, bool check )
 {
+    // Prevent the emission of wxEVT_CHECKLISTBOX event by temporarily block all
```

Comment:
Sorry, another nitpick:
```suggestion
    // Prevent the emission of wxEVT_CHECKLISTBOX event by temporarily blocking all
```